src/gpu/generic/sycl/ref_matmul.cpp

```diff
@@ -52,16 +55,56 @@ void ref_matmul_t::pd_t::init_conf() {
     for (const auto &mdw : {src_d, weights_d, dst_d, bias_d}) {
         if (mdw.has_runtime_dims()) {
             any_runtime_params_ = true;
-            return;
+            return status::unimplemented;
```

Comment:
(minor) please  don't hesitate to use [verbose macros](https://github.com/oneapi-src/oneDNN/blob/0bff5bd0f56dcf5e68303198996e15a065ffb16d/src/common/verbose.hpp#L87-L159) for this kind of checks.

Reply:
Great suggestion, made the change and prefer it this way as well, thank you